README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
Show/hide variable only works with a sleep. The program below will not show the variable. And even if you add a sleep after the "show variable" block, the variable is not shown until the end of the sleep period.

<img width="85" alt="image" src="https://user-images.githubusercontent.com/126523/222972104-ba90adc9-ad4d-4f11-8abe-a44291981938.png">

```
<xml xmlns="http://www.w3.org/1999/xhtml">
  <variables>
    <variable type="" id="!BBtuLB9KEV*u}`|z*2@" islocal="false" iscloud="false">test</variable>
    <variable type="" id="6fz|yAaemr1QU;[@_q@s" islocal="false" iscloud="false">i</variable>
    <variable type="" id="!bq%PTcdJnN1HU9`d$4Q" islocal="false" iscloud="false">j</variable>
    <variable type="" id="0-XOGI~a{yiQ_SB.$7fp" islocal="false" iscloud="false">k</variable>
    <variable type="" id="Xm?:86*Zayd!O*R5xR#:" islocal="false" iscloud="false">m</variable>
    <variable type="" id="=g$Rw,tGqz7QYDTx]UVJ" islocal="false" iscloud="false">n</variable>
  </variables>
  <block type="control_run" id="JorN:3{p^ujG@Qr`(,y#" x="137" y="153">
    <next>
      <block type="data_setvariableto" id="}IUiky{mG@/EbyhQ]gpD">
        <field name="VARIABLE" id="!BBtuLB9KEV*u}`|z*2@" variabletype="">test</field>
        <value name="VALUE">
          <shadow type="text" id="5qjD9hI6?wN$FT}]PUl]">
            <field name="TEXT">2</field>
          </shadow>
        </value>
        <next>
          <block type="data_showvariable" id="hbW#3fpt%yqf2AXyoT-l">
            <field name="VARIABLE" id="!BBtuLB9KEV*u}`|z*2@" variabletype="">test</field>
          </block>
        </next>
      </block>
    </next>
  </block>
</xml>
```